src/app/adapters/adapterSet.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import {Graph, type NodeAddressT, type EdgeAddressT} from "../../core/graph";
+import {NodeTrie, EdgeTrie} from "../../core/trie";
+import type {Repo} from "../../core/repo";
+
+import type {
+  StaticPluginAdapter,
+  DynamicPluginAdapter,
+  NodeType,
+  EdgeType,
+} from "./pluginAdapter";
+
+import {FallbackStaticAdapter} from "./fallbackAdapter";
+
+export class StaticAdapterSet {
+  _adapters: $ReadOnlyArray<StaticPluginAdapter>;
+  adapterNodeTrie: NodeTrie<StaticPluginAdapter>;
```

Comment:
Can you underscore-name all of these to be clear that they are all private?

Reply:
Done.